src/LINEBot/MessageBuilder/Flex/ComponentBuilder/BoxComponentBuilder.php

```diff
@@ -46,6 +47,41 @@ class BoxComponentBuilder implements ComponentBuilder
     /** @var TemplateActionBuilder */
     private $actionBuilder;
 
+    /** @var string */
+    private $paddingAll;
```

Comment:
For the properties added this time, we are only considering sets via setters.
Because I thought it would be confusing to users because there were too many parameters when trying to specify it in the constructor.